Makefile

```diff
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.13.3
```

Comment:
@parabolic nit: we should pin to minor image version `0.13` so this can automatically get patch updates when `0.13.4,0.13.5` etc releases.